sources/procfs.go

```diff
@@ -528,8 +528,11 @@ func getJobStatsIOMetrics(jobBlock string, jobID string, promName string, helpTe
 
 func getJobNum(jobBlock string) (jobID string, err error) {
 	jobID = regexCaptureString("job_id: .*", jobBlock)
-	jobID = regexCaptureString("[0-9]*\\.[0-9]+|[0-9]+", jobID)
-	return strings.Trim(jobID, " "), nil
+	jobIDlist := regexCaptureNumbers(jobID)
+	if len(jobIDlist) < 1 {
```

Comment:
I intended it to be `< 1` here. The `job_id: XXXX` line should always be of that format and there should only be one number which ends the line. If for some reason there is another number at the end, I believe we can safely say it would not be a part of the Job ID, though I don't see a scenario where this would ever be the case. In short, we don't really care if it is longer, just as long as there is at least one number.